Globes.js

```diff
@@ -0,0 +1,94 @@
+{
+	"translatorID": "4e9c34ef-89bb-4c7e-97dd-abb61d8b9254",
+	"label": "Globes",
+	"creator": "Eran Rosenthal",
+	"target": "^https?://www\\.globes\\.co\\.il/news/article.aspx",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2019-10-01 10:49:38"
+}
+
+/**
+	Copyright (c) 2019 Eran Rosenthal
+	
+	This program is free software: you can redistribute it and/or
+	modify it under the terms of the GNU Affero General Public License
+	as published by the Free Software Foundation, either version 3 of
+	the License, or (at your option) any later version.
+	
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+	Affero General Public License for more details.
+	
+	You should have received a copy of the GNU Affero General Public
+	License along with this program. If not, see
+	<http://www.gnu.org/licenses/>.
+*/
+
+function detectWeb(doc, url) {
+	return 'newspaperArticle';
+}
+function doWeb(doc, url) { 
+	var item = new Zotero.Item('newspaperArticle');
+	
+	item.title = ZU.xpathText(doc, '//meta[@property="og:title"]/@content');
+	item.publicationTitle = 'Globes';
+	item.url = url;
+	item.language = 'he';
+	var abstract = ZU.xpathText(doc, '//meta[@name="description"]/@content');
+	if (abstract) {
+		item.abstractNote = abstract;
+	}
+	
+	var json = ZU.xpathText(doc, '//script[@type="application/ld+json"]');
+	var data = JSON.parse(json);
+	if (data) {
+		
+		if (data.author && Array.isArray(data.author)) {
+			// note only the 0 item is a real author
+			item.creators.push(Zotero.Utilities.cleanAuthor(data.author[0].name, 'author'));
```

Comment:
How about articles with multiple authors? I guess we should restrict this in another way...

Reply:
added to test cases few different cases - one where multiple "authors" are real author, and date and one where there are really 2 authors.